logging/bin/deploy_logging_open.sh

```diff
@@ -86,7 +86,7 @@ set -e
 # Version Info                   #
 ##################################
 
-if ! deployV4MInfo "$LOG_NS"; then
+if ! deployV4MInfo "$LOG_NS" "v4m-logging"; then
```

Comment:
I'm going to suggest we use "logs" rather than "logging".  Similarly, let's use "metrics" instead of "monitoring" and "traces" when we add support for distributed tracing.